src/app/pages/session/edit/[id]/index.vue

```diff
@@ -87,72 +135,49 @@
           </CardButton>
         </div>
       </section>
-      <section class="flex flex-col gap-4">
-        <span class="text-xl font-bold">{{ t('metadata') }}</span>
-        <div class="flex flex-col gap-2">
-          <CardButton
-            :description="t('aboutDescription')"
-            :title="t('about')"
-            :to="localePath('index')"
-          >
-            <IHeroiconsNewspaper />
-          </CardButton>
-          <!-- <CardButton
-            :description="t('teamDescription')"
-            :title="t('team')"
-            :to="localePath('/team')"
-          >
-            <IHeroiconsUserGroup />
-          </CardButton> -->
-          <CardButton
-            :description="t('statusDescription')"
-            :is-external="true"
-            :title="t('status')"
-            to="https://status.maev.si/"
-          >
-            <IHeroiconsServerStack />
-          </CardButton>
-          <CardButton
-            :description="t('sourceCodeDescription')"
-            :is-external="true"
-            :title="t('sourceCode')"
-            to="https://github.com/maevsi/"
-          >
-            <IHeroiconsCodeBracket />
-          </CardButton>
-          <CardButton
-            :description="t('developerInformationDescription')"
-            :title="t('developerInformation')"
-            :to="`/session/view/${route.params.id}`"
-          >
-            <IHeroiconsWrench />
-          </CardButton>
```

Comment:
Please leave this button in, at least until we actually release, as we need a way to access the testing page.